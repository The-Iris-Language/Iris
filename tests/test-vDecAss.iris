.. test declarations and assignment on same line
class Main () {
  public:
    int main() {
      int i = 0;
      int j;

      i = 1;
      
      int k;
      for (int i = 0; i < 10; i++) {
        int w;
      }
      return 0;
    }
}