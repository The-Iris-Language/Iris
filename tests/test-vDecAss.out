class Main() {
int main() {
int i = 0;
int j;
i = 1;
int k;
for (int i = 0; i < 10; i++) {
    int w;
}
return 0;
}
}